src/WebJobs.Script/Config/ScriptJobHostOptionsSetup.cs

```diff
@@ -59,6 +60,12 @@ public void Configure(ScriptJobHostOptions options)
             // FunctionTimeout
             ConfigureFunctionTimeout(options);
 
+            if (_environment.IsLogicApp())
```

Comment:
Wondering if we should default to infinite, but honor if explicitly configured (i.e., keep the value if set in config)

Reply:
Don't we still want to have a timeout set for general customers?